ohapi/api.py

```diff
@@ -132,17 +132,14 @@ def upload_file(target_filepath, metadata, access_token, project_member_id,
                          'file size'.format(target_filepath))
             return
 
-    url = urlparse.urljoin(
-        base_url, '/api/direct-sharing/project/files/upload/?{}'.format(
-            urlparse.urlencode({'access_token': access_token})))
+    # url = urlparse.urljoin(base_url, '/api/direct-sharing/project/files/upload/?{}'
+    # .format(urlparse.urlencode({'access_token': access_token})))
 
     logging.info('Uploading {} ({})'.format(
         target_filepath, format_size(filesize)))
 
-    r = requests.post(url,
-                      files={'data_file': open(target_filepath, 'rb')},
-                      data={'project_member_id': project_member_id,
-                            'metadata': json.dumps(metadata)})
+    # r = requests.post(url, files={'data_file': open(target_filepath, 'rb')},
```

Comment:
This is removing functionality: `requests` is performing an action here, but storing the result is optional. So, to make `flake8` happy, the only thing you should remove is `r = `. Please update to fix this, thanks!